cloudshell/shell/core/driver_context.py

```diff
@@ -55,6 +55,9 @@ class ConnectivityContext:
     :type cloudshell_version: str
 
     :type cloudshell_api_scheme: str
+
+    :param use_webapi_endpoint: Is Cloudshell API Web Api Endpoint enabled on server?
```

Comment:
As I understand the value can be "True" or "False", right? Can you mention this in the comment, please

Reply:
good point!